tests/utils_tests.cpp

```diff
@@ -50,17 +47,17 @@ BOOST_AUTO_TEST_CASE(CheckSumTest)
 /**
  * Simple test for PathExists.
  */
-BOOST_AUTO_TEST_CASE(PathExistsTest)
+TEST_CASE("PathExistsTest", "[UtilsTest]")
 {
   // Check for files that exist.
-  BOOST_REQUIRE(Utils::PathExists("./../tests/CMakeLists.txt"));
-  BOOST_REQUIRE(Utils::PathExists("./../CMakeLists.txt"));
+  REQUIRE(Utils::PathExists("./../tests/CMakeLists.txt") == true);
```

Comment:
Depending on the current directory, this could fail, can you change the path to `./../../tests/CMakeLists.txt"`, and see if that works.

Reply:
Yes sure, will do that.